ui/src/app/views/profile-page/component.spec.ts

```diff
@@ -76,11 +78,46 @@ describe('ProfilePageComponent', () => {
     }).compileComponents().then(() => {
       fixture = TestBed.createComponent(ProfilePageComponent);
       tick();
+      updateAndTick(fixture);
     });
   }));
 
   it('should render', fakeAsync(() => {
     updateAndTick(fixture);
     expect(fixture).toBeTruthy();
   }));
+
+  it('handles long given name errors', fakeAsync(() => {
+    console.log(fixture);
+    simulateInput(
+      fixture, fixture.debugElement.query(By.css('#givenName')), randomString(81));
+    tick(300);
+    updateAndTick(fixture);
+    expect(fixture.componentInstance.givenNameValid).toBeFalsy();
```

Comment:
If there is a way, I would probably recommend making sure the error element shows up on the page, rather than just reading the component variable. (From a test philosophy standpoint)

Same comment for the following tests.

Reply:
Done.